app/forms/questionnaire_form.py

```diff
@@ -200,15 +200,17 @@ def validate_date_range_with_period_limits_and_single_date_limits(
         period_range: timedelta,
     ) -> None:
         # Get period_limits from question
-        period_limits_item: tuple = self._get_period_limits(period_limits)
-        period_min: Mapping[str, int] = period_limits_item[0]
-        min_offset = self._get_offset_value(period_min)
+        period_limits_item = self._get_period_limits(period_limits)
+        period_min = period_limits_item[0]
 
-        # Exception to be raised if range available is smaller than minimum range allowed
-        if period_min and period_range < min_offset:
-            exception = f"The schema has invalid period_limits for {question_id}"
+        if period_min:
+            min_offset = self._get_offset_value(period_min)
 
-            raise Exception(exception)
+            # Exception to be raised if range available is smaller than minimum range allowed
+            if period_range < min_offset:
```

Comment:
What if they are equal ?

Reply:
Its the minimum allowed so is fine as doesn't need an exception